### DISCUSSION TITLE
2.6 for mac?

### DISCUSSION THREAD

**Discussion by MonsterParties** [Dpeta/pesterchum-alt-servers#172] [General]
Hi, I'm a mac user--when might the latest update be available for mac? Absolutely no pressure lol --I'm just curious, I cant emulate the windows version on wine because my storage is too small. The changes look really cool and useful, I'm excited to use them!

**Comment by Dpeta** [comment]
haii sorry I'll probably get around to it soon o7, mac issues are just a bit hard to test rn since I don't have a vm set up
